README.md

```diff
@@ -86,28 +86,9 @@ Premium package:
 
 - [`@mui/x-data-grid-premium`](https://www.npmjs.com/package/@mui/x-data-grid-premium)
 
-## Questions and feedback
+## Support
 
-### GitHub
-
-We use GitHub issues as a bug and feature request tracker.
-If you think you've found a bug, or you have a new feature idea, please start by [making sure it hasn't already been reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed).
```

Comment:
Remove all of that, I don't see the value of it, it looks best to link to the docs, to have 1. no duplication 2. something always up to date, even on older npm packages releases.